test/k8sT/Services.go

```diff
@@ -2760,6 +2760,26 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`, helpers.DualStackSupp
 					testNodePort(true, false, false, 0) // no need to test from outside, as testDSR did it
 				})
 
+				It("Should not drop SCTP packets", func() {
```

Comment:
I think we only want to run this test if we're running our kube-proxy replacement, no? (According to https://github.com/cilium/cilium/issues/10541.)